hack/cve-triage/cve_triage_core.py

```
@@ -0,0 +1,2288 @@
+                print(f"     The golang toolset team may not have filed it yet.")
+                print(f"     Wait for their ticket, then re-run to complete deduplication.")
+        return 1
+
+    # ------------------------------------------------------------------
+    # Step 3: Verify govulncheck is aware of the CVE
+    # ------------------------------------------------------------------
+    print(f"\n[3] Checking govulncheck DB awareness...")
+    advisory_freshness = (
+        (go_data or {}).get("db_modified") or (go_data or {}).get("published")
+    )
+    aware, db_modified_ver, scanner_ver, warn = check_govulncheck_awareness(
```

Ruff (0.16.1)

[warning] Unpacked variable `aware` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)